src/actions-on-google-ava.ts

```diff
@@ -16,7 +16,7 @@
 
 // Implementation of Google Assistant with Ava testing framework wrapping
 
-import test, { GenericCallbackTestContext } from 'ava'
+import test, { ExecutionContext } from 'ava'
```

Comment:
Would it make more sense to break this into a PR to update Ava (if it supports Node 6) and a second PR to add the new modes?

Reply:
ava has required Node 8 since v2.0.0 -- as have many npm libraries since Node 6.x releases reached end of life on April 30th

Reply:
Since Node 6 has reached end of life, many npm libraries have dropped support. I have split out the new modes into separate PRs -- perhaps a last release on the 0.2.x train before switching to 0.3.x given the new Node version?